webdriverio/web/webdriver/test/configs/wdio.saucelabs.ios.conf.js

```diff
@@ -0,0 +1,98 @@
+const {config} = require('./wdio.shared.conf');
+const build = `iOS Safari WebdriverIO-V6 build-${new Date().getTime()}`
+
+// =========================
+// Sauce Simulator specific config
+// =========================
+config.user = process.env.SAUCE_USERNAME;
+config.key = process.env.SAUCE_ACCESS_KEY;
+// If you run your tests on Sauce Labs you can specify the region you want to run your tests
+// in via the `region` property. Available short handles for regions are `us` (default) and `eu`.
+// These regions are used for the Sauce Labs VM cloud and the Sauce Labs Real Device Cloud.
+// If you don't provide the region, it defaults to `us`.
+config.region = process.env.REGION || 'us';
+
+// ============
+// Capabilities
+// ============
+config.capabilities = [
+    /**
+     * iOS Safari
+     */
+    {
+        name: 'iOS 13.0 iPhone safari',
```

Comment:
Hmm, I wonder if we should cover all OS versions here, reason for that is that it can change over time, my suggestion would be to pick 1 iOS `12`, `13` and `14` version and keep it like that. 
Adding the link to the platform configurator in the comments and mention they can construct it there would be sufficient.

Reply:
done!